### PR TITLE
Option for shorter button text in Telegram

### PR DESCRIPTION
**Pull request by fseimb**
Added short button text as part of the messenger mode option. If messenger mode is set to true, Telegram will display fewer buttons with shorter button text.

### CODE REVIEW

File: config/bot_token.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-BOT_TOKEN: <<token>>
-MESSENGER_TOKEN: <<token>>
+BOT_TOKEN: "1677935498:AAFnSl2FXPaSZ8xlNAI2CzTGvaiQNBOj89Y"
+MESSENGER_TOKEN: "EAAF5ZA8L6hnUBAJjhWUh5g0sk7ZB6Ub9yKuEzk6Ael2Ym5oX2JpTSJHT0rk9WAVGNw92ZB3pzkNMfw5PnTZA7SpVqa0fTLAwWZCQvK4I0npb1ZB9zIqZAKqRuUQheIVglOlqw4mZCP3ZApvVKxYsAymDCn1ZCK7iqsTp92CHYBGwbGZA57B14S9rgOl"
```
Comment (kbalog) [inline config/bot_token.yaml:2]: These should not be pushed

File: config/bot_token.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 MESSENGER_TEST_TOKEN: <<token>>
-MESSENGER_VERIFY_TOKEN: <<token>>
\ No newline at end of file
+MESSENGER_VERIFY_TOKEN: "bonobo"
```
Comment (kbalog) [inline config/bot_token.yaml:4]: This should not be pushed

File: moviebot/controller/server.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 app = Flask(__name__)
 controller_messenger = ControllerMessenger()
 controller_telegram = ControllerTelegram()
-URL = "https://636d944311f9.ngrok.io/" # Webhook url
+URL = "https://025d8e8dc5c6.ngrok.io/" # Webhook url
```
Comment (kbalog) [inline moviebot/controller/server.py:13]: Move this to config file or pass it as a command line argument

**Review by kbalog — CHANGES_REQUESTED** [review]

**Comment by IKostric** [comment]
@kbalog Should this PR be followed up on or closed?

**Comment by kbalog** [comment]
Moving the server address and tokens to a config file should be done. Not sure the messenger controller and other changes in this PR are needed.

**Pull request closed without merging**